dist/AccordionSection/AccordionHeader.d.ts

```diff
@@ -7,7 +7,7 @@ declare function AccordionHeader({ isOpen, note, onClick, panelId, text, iconRig
     panelId: any;
     text: any;
     iconRight: any;
-}): JSX.Element;
+}): import("@emotion/react/jsx-runtime").JSX.Element;
```

Comment:
This is due to `"jsxImportSource": "@emotion/react"` in `tsconfig.json`, but ultimately this is effectively an alias for the same type in React.